airflow/models/connection.py

```diff
@@ -35,6 +36,7 @@
 from airflow.utils.module_loading import import_string
 
 log = logging.getLogger(__name__)
+_RE_COMPILED = re.compile("^[\w\@\#\$\%\&\!\(\)\*\-]{1,200}$")
```

Comment:
Can we name this a bit better to explain what the pattern tries to do?

Reply:
definitely, I'll do that now.

Reply:
@uranusjr just updated the PR with improvements in both docs and name.